src/onnx/parse_convolution.cpp

```diff
@@ -149,9 +149,40 @@ struct parse_convolution : op_parser<parse_convolution>
 
         recalc_conv_attributes(values, kdims);
 
+        instruction_ref ret;
+        // parse a_zero_point and b_zero_point values
+        auto l0_zp = l0;
+        auto w_zp =  weights;
+
         op.from_value(values);
-        auto l1 = info.add_instruction(op, l0, args[1]);
-        return info.add_bias(args, l1, 1);
+        if (op.name() == "quant_convolution") {
+            if(args.size() > 2)
+            {
+                l0_zp = args[2];
+                if (l0_zp->get_shape().type() != l0_shape.type()) {
+                    MIGRAPHX_THROW("PARSE: ConvInteger Data and Data Zero Point must have same type");
+                }
+
+                l0_zp = info.add_common_op("sub", l0, l0_zp);
+
+                if(args.size() > 3)
+                {
+                    w_zp = args[3];
```

Comment:
Are we handling both cases where w_zero_point is a scalar or a tensor..? Thanks.

Reply:
We need to as per onnx spec. I can handle that

Reply:
Should be handled now. I've added common op on the subtract to the bias to handle this.